node/full_node_test.go

```diff
@@ -9,52 +9,37 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
-	"github.com/stretchr/testify/require"
 
 	abci "github.com/cometbft/cometbft/abci/types"
-	"github.com/cometbft/cometbft/libs/log"
-	proxy "github.com/cometbft/cometbft/proxy"
-	"github.com/cometbft/cometbft/types"
 	"github.com/libp2p/go-libp2p/core/crypto"
 	"github.com/libp2p/go-libp2p/core/peer"
 
 	testutils "github.com/celestiaorg/utils/test"
 
-	"github.com/rollkit/rollkit/config"
 	"github.com/rollkit/rollkit/mempool"
 	"github.com/rollkit/rollkit/test/mocks"
 )
 
 // TestStartup checks if the node starts and stops without any errors
 func TestStartup(t *testing.T) {
-	assert := assert.New(t)
-	require := require.New(t)
-
 	ctx := context.Background()
-	node := initializeAndStartNode(ctx, require, assert)
-	defer cleanUpNode(node, assert)
+	node := initializeAndStartFullNode(ctx, t)
+	defer cleanUpNode(node, t)
 }
 
 func TestMempoolDirectly(t *testing.T) {
-	assert := assert.New(t)
-	require := require.New(t)
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
 
-	ctx := context.Background()
-	node := initializeAndStartNode(ctx, require, assert)
-	defer cleanUpNode(node, assert)
+	node := initializeAndStartFullNode(ctx, t)
+	assert := assert.New(t)
```

Comment:
Instead of initializing `assert` here, could we instead import `	"github.com/stretchr/testify/assert"` just like in the new implementation of `cleanUpNode`

Reply:
The reason I didn't touch that was to manage scope creep on this PR. 
In general I think it is cleaner and more flexible to pass the `testing.T` around vs `assert` or `require` but I didn't want to edit `gerPeerID` and the verify helpers in this PR. 

This could be a follow up if we wanted, but it is pretty low priority to do this type of testing clean up so we should probably just leave it for now.

Reply:
Created a follow-up issue: https://github.com/rollkit/rollkit/issues/1320